SECURITY.md

```diff
@@ -1,6 +1,75 @@
-# Reporting Security Issues
+# Security Policy
 
-If you believe you have found a security vulnerability in PyTorch, we encourage you to let us know right away. We will investigate all legitimate reports and do our best to quickly fix the problem.
+ - [**Using Pytorch Securely**](#using-pytorch-securely)
+   - [Untrusted models](#untrusted-models)
+   - [Untrusted inputs](#untrusted-inputs)
+   - [Data privacy](#data-privacy)
+   - [Untrusted environments or networks](#untrusted-environments-or-networks)
+   - [Multi-Tenant environments](#multi-tenant-environments)
+ - [**Reporting a Vulnerability**](#reporting-a-vulnerability)
+
+## Using Pytorch Securely
+### Untrusted models
```

Comment:
Should it be prefaces with "PyTorch models are programs" similar to https://github.com/tensorflow/tensorflow/blob/master/SECURITY.md#tensorflow-models-are-programs

And do you mind adding a paragraph that one way to mitigate the risk is to always separate model and weights definitions by using `weights_only` option to `torch.load` or `.safetensors` format?

Reply:
Great suggestion! I've edited the PR incorporating those ideas, PTAL.